internal/daemon/controller/controller.go

```diff
@@ -218,6 +218,7 @@ func New(ctx context.Context, conf *Config) (*Controller, error) {
 		ctx,
 		kms.WithRootWrapper(c.conf.RootKms),
 		kms.WithWorkerAuthWrapper(c.conf.WorkerAuthKms),
+		kms.WithWorkerStorageWrapper(c.conf.WorkerStorage),
```

Comment:
WorkerStorage is only set on the worker config/ used by the worker, so the controller doesn't need it
```suggestion
	
```